packages/caliper-fabric/test/connector-versions/v1/WalletFacade.js

```diff
@@ -0,0 +1,141 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+chai.should();
+const mockery = require('mockery');
+
+/**
+ * simulate a node sdk v1 wallet
+ */
+class StubWallet {
+    /**
+     * Mock V1 wallet implementation
+     */
+    constructor() {
+        this.map = new Map();
+    }
+
+    /**
+     *
+     * @param {*} key b
+     * @param {*} value b
+     */
+    async import(key, value) {
+        this.map.set(key, value);
+    }
+
+    /**
+     *
+     * @param {*} key b
+     */
+    async export(key) {
+        return this.map.get(key);
+    }
+
+    /**
+     *
+     */
+    async list() {
+        return Array.from(this.map.keys());
+    }
+}
+
+/**
+ * InMemoryWallet copy
+ */
+class InMemoryWallet extends StubWallet {}
+
+/**
+ * FileSystemWallet copy
+ */
+class FileSystemWallet extends StubWallet {}
+
+/**
+ * x509walletmixin class
+ */
+class X509WalletMixin {
+    /**
+    * @param {*} mspid b
+    * @param {*} cert b
+    * @param {*} key b
+    * @return {*} identity b
+    */
+    static createIdentity(mspid, cert, key){
+        const identity = {
+            credentials: {
+                cert,
+                key
+            },
+            mspid,
+            type: 'X.509',
+        };
+        return identity;
+    }
+}
+
+mockery.enable();
+mockery.registerMock('fabric-network',  {FileSystemWallet, InMemoryWallet, X509WalletMixin});
+
+const WalletFacadeFactory = require('../../../lib/connector-versions/v1/WalletFacadeFactory');
+const WalletFacade = require('../../../lib/connector-versions/v1/WalletFacade');
+
+describe('When testing a V1 Wallet Facade Implementation', () => {
+
+    beforeEach(() => {
+        mockery.enable();
+        mockery.registerMock('fabric-network',  {FileSystemWallet, InMemoryWallet, X509WalletMixin});
+    });
+
+    afterEach(() => {
```

Comment:
change this from `afterEach` to `after` by doing this it means the beforeEach can be removed